.github/infrastructure/conformance/azure/setup-azure-conf-test.sh

```diff
@@ -415,3 +415,9 @@ echo "INFO: setup-azure-conf-test completed."
 echo "INFO: Remember to \`source ${ENV_CONFIG_FILENAME}\` before running local conformance tests."
 echo "INFO: ${AZURE_CREDENTIALS_FILENAME} contains the repository secret to set to run the GitHub conformance test workflow."
 echo "INFO: To teardown the conformance test resources, run ${TEARDOWN_SCRIPT_NAME}."
+
+# ----------------------------------
+# Populate SQLServer test settings
+# ----------------------------------
+echo "Configuring SQL Server test settings ..."
+echo export DAPR_TEST_SQL_CONNSTRING="server=localhost;user id=sa;password=Pass@Word1;port=1433;" >> "${ENV_CONFIG_FILENAME}"
```

Comment:
Is there any places to control those hardcode value? I prefer to set and use them as ENV.

Reply:
This line isn't necessary anymore for the conformance test because I hardcoded it to the Docker container connection string anyway.

But note that as of 2 years ago or so this environment variable is used to run SQL Server integration tests (take a look at `state/sqlserver/sqlserver_integration_test.go`-- I just used whatever was already being used there.

Of course ultimately you can always configure conformance tests differently by editing values in `tests/conformance/config`. The environment variables exposed there exist for our CI / GitHub actions purposes. So if you want to use a different environment variable yourself, `tests/conformance/config` is the place to do that.